.eslintrc.js

```diff
@@ -178,6 +178,13 @@ module.exports = {
     'prefer-arrow-callback': 'error',
     'prefer-const': ['error', {'ignoreReadBeforeAssign': true}],
     // }}}
+
+    'max-len': ['error', {
+      'code': 120,
```

Comment:
Sorry, but we follow the Google JS style guide, which limits line length to 80 characters.  I don't think we should change that project-wide just for the TS defs generator.

If you need to override that in certain places, you can use a comment like:

```js
// eslint-disable-next-line max-len
```